apps/extension/src/ui/domains/Portfolio/AssetDetails/SendFundsIconButton.tsx

```diff
@@ -10,23 +10,32 @@ import { isTransferableToken } from "@ui/util/isTransferableToken"
 import { usePortfolioNavigation } from "../usePortfolioNavigation"
 
 export const SendFundsButton = ({
-  symbol,
   networkId,
   shouldClose,
+  ...tokenLookup
 }: {
-  symbol: string
   networkId: ChainId | EvmNetworkId
   shouldClose?: boolean
-}) => {
+} & ({ tokenId: string } | { symbol: string })) => {
+  const tokenId = "tokenId" in tokenLookup ? tokenLookup.tokenId : undefined
+  const symbol = "symbol" in tokenLookup ? tokenLookup.symbol : undefined
+
   const { selectedAccount } = usePortfolioNavigation()
   const [includeTestnets] = useSetting("useTestnets")
   const tokens = useTokens({ activeOnly: true, includeTestnets })
 
   const token = tokens?.find(
-    (t) =>
-      t.symbol === symbol &&
-      isTransferableToken(t) &&
-      (("evmNetwork" in t && t.evmNetwork?.id === networkId) || t.chain?.id === networkId),
+    typeof tokenId === "string"
+      ? // search by tokenId
+        (t) =>
+          t.id === tokenId &&
+          isTransferableToken(t) &&
+          (("evmNetwork" in t && t.evmNetwork?.id === networkId) || t.chain?.id === networkId)
+      : // search by token symbol
+        (t) =>
+          t.symbol === symbol &&
+          isTransferableToken(t) &&
+          (("evmNetwork" in t && t.evmNetwork?.id === networkId) || t.chain?.id === networkId),
   )
 
   const { canSendFunds, cannotSendFundsReason, openSendFundsPopup } = useSendFundsPopup(
```

Comment:
`useSendFundsPopup` takes `tokenSymbol` as a prop, this suggests two things:
1. We should stick with the same naming convention for props (so call it `tokenSymbol` instead of `symbol` in the props for this hook)
2. rather than performing a `find` if the token symbol is given, and then giving the first token found to the hook, we should just pass the token symbol to the hook directly, and allow the logic in the hook to determine which token/s to use